app/controllers/admin/registrations_controller.rb

```diff
@@ -6,12 +6,17 @@ class RegistrationsController < Admin::BaseController
 
     def index
       authorize! :show, Registration.new(conference_id: @conference.id)
-      @pdf_filename = "#{@conference.title}.pdf"
-      @registrations = @conference.registrations.includes(:user).order('registrations.created_at ASC')
-      @attended = @conference.registrations.where('attended = ?', true).count
+      @registrations = @conference.registrations.eager_load(
```

Comment:
why `eager_load` instead of `includes`? :thinking:

Reply:
`eager_load` instantiates the dependent object and does so in a _single_ query; `includes` adds additional queries for each included model.

http://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations